ui/cypress.json

```diff
@@ -15,5 +15,6 @@
     "mochaFile": "junit/cypress-junit.xml",
     "toConsole": true
   },
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "baseUrl": "http://localhost:3000"
```

Comment:
Maybe we can get rid of `env.target_url` then?

Reply:
Well, I am not sure we want to remove it. The cost of removing it would be hardcode this `cy.visit(http://localhost:3000)` all the times.

Reply:
?? Isn't the point of `baseUrl` to always prefix any path for `cy.visit`?
We would just need to replace any `cy.visit(Cypress.env('target_url'))` by `cy.visit('/')`.

Reply:
Indeed!! ;)